fragments/app-sso-client/accelerator.yaml

```diff
@@ -1,71 +1,74 @@
 accelerator:
   options:
-    - name: workloadName
+    - name: appssoOfferingName
+      description: To find the available AppSSO service offerings in your cluster, run `tanzu services classes list`.
       inputType: text
-      label: Workload name
-      required: true
-    - name: workloadURL
-      inputType: text
-      label: Workload URL
-      description: "A fully qualified URL the app will be available on including a protocol (e.g 'https://{application-artifact-name}.{namespace}.{tapclusterdomain}')"
-      required: true
-    - name: appSsoServerLabel
-      inputType: text
-      label: AppSSO server label
-      description: "Enter the key: value pair that is the identifying label on Authserver in the form of  'identifier: authserver-identifier' (e.g. 'name: authserver')"
-      required: true
+      label: AppSSO offering name
 
 engine:
   merge:
-    - include: [ "**/client-registration.yaml", "**/client-registration-claim.yaml" ]
-      chain:
-        - type: ReplaceText
-          substitutions:
-            - text: "<my-workload>"
-              with: "#workloadName"
-            - text: "<my-workload-url>"
-              with: "#workloadURL"
-            - text: "<auth-server-selector>"
-              with: "#appSsoServerLabel"
     - include: [ "**/workload.yaml" ]
       chain:
         - type: OpenRewriteRecipe
           recipe: org.openrewrite.yaml.MergeYaml
           options:
             key: "'$.spec'"
-            yaml: "'serviceClaims:\n  - name: ' + #workloadName + '\n    ref:\n      apiVersion: services.apps.tanzu.vmware.com/v1alpha1\n      kind: ResourceClaim\n      name: ' + #workloadName + '-client-registration-claim'"
+            yaml: "'serviceClaims:\n  - name: ' + #workloadName + '\n    ref:\n      apiVersion: services.apps.tanzu.vmware.com/v1alpha1\n      kind: ClassClaim\n      name: ' + #workloadName "
```

Comment:
**[suggestion, non-blocking]:** Wouldn't a [block scalar](https://yaml-multiline.info/) be more readable here? I don't think it will break anything, but I can't say.